core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/WorkspaceContext.scala

```diff
@@ -1,14 +0,0 @@
-package org.broadinstitute.dsde.rawls.dataaccess
-
-import java.util.UUID
-
-import org.broadinstitute.dsde.rawls.model.Workspace
-
-/**
- * Holds information about a workspace inside a DB transaction.
- *
- * @param workspace
- */
-case class SlickWorkspaceContext(workspace: Workspace) {
```

Comment:
there are a couple commented-out tests that refer to `SlickWorkspaceContext`. Do we care about these / is it worth updating them? If not, then maybe we just delete them?